src/server.ts

```diff
@@ -2,16 +2,25 @@ import { ApolloServer, gql } from 'apollo-server';
 
 const typeDefs = gql`
   type Query {
-    hello: String,
+    hello: String
   }
 `;
 
 const resolvers = {
   Query: {
-    hello: () => 'hello, world!',
+    hello: () => {
+      return 'hello, world!';
+    },
   },
-}
+};
 
 const server = new ApolloServer({ typeDefs, resolvers });
 
-server.listen().then(({ url }) => console.log(`server is running on ${url}`))
+server
+  .listen()
+  .then(({ url }) => {
+    console.log(`server is running on ${url}`);
+  })
+  .catch(() => {
+    console.log(`Something went wrong.`);
+  });
```

Comment:
se retornar um objeto de erro, seria uma boa colocar a msg dele. algo assim 
```suggestion
  .catch((error) => {
    console.log(`Something went wrong. ${error?.message}`);
  });
```
o que acha?

Reply:
Fiz algo semelhante, pois o ESLint deu alguns erros da forma como você sugeriu.